.github/workflows/deploy.yml

```diff
@@ -2,11 +2,12 @@ name: Deploy Website to Github Pages
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
+  pull_request:
```

Comment:
So this means it will trigger for each pull request made, right?

Reply:
correct! You can see that it actually triggered for this PR (and that the deploy step was skipped).